app/models/work.rb

```diff
@@ -120,8 +120,13 @@ def all_descendent_members
   # All DISPLAYABLE (to current user) members, in order, and
   # with proper pre-fetches.
   def ordered_viewable_members(current_user:)
+
+
     members = self.members.includes(:leaf_representative)
-    members = members.where(published: true) if current_user.nil?
+
+    show_unpublished = AccessPolicy.new(current_user).can_see_unpublished_records?
+    members = members.where(published: true) unless show_unpublished
```

Comment:
just make this an if/unless instead of using a temporary local variable

Reply:
Done!